src/EOS.hpp

```diff
@@ -15,6 +15,16 @@
 #include "AMReX_REAL.H"
 #include "physics_info.hpp"
 
+#include "burn_type.H"
+#include "eos.H"
+#include "extern_parameters.H"
+
+#ifdef PRIMORDIAL_CHEM
+constexpr bool PRIMORDIAL_CHEM_ENABLED = true;
+#else
+constexpr bool PRIMORDIAL_CHEM_ENABLED = false;
+#endif
+
```

Comment:
Since this code is not specific to primordial chemistry, can we change the variable names to just `CHEMISTRY_ENABLED` or `CHEM_ENABLED`?

Reply:
In an earlier PR, we decided to continue using PRIMORDIAL_CHEM instead of CHEM. Once we are finished with all the code development for Pop III simulations, I will then systematically go through each file and make the changes. So, it's better to keep it PRIMORDIAL_CHEM for now and make the change everywhere at the same time in the end .

Reply:
Assume that I don't remember any earlier PRs. My memory is wiped when I click 'merge' :)